### PR TITLE
Test the new Podfile syntax for handling UIFonts with a single repo

### PR DESCRIPTION
**Pull request by orta**
Sibling PR to https://github.com/artsy/Artsy-OSSUIFonts/pull/7

Should be the main part of the fixes for https://github.com/artsy/eigen/issues/2088

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">No CHANGELOG changes made</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is classed as Work in Progress</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by orta** [comment]
👍 This verifies that the changes don't break our stuff - and it's definitely using the new repo

screen shot 2017-01-18 at 11 25 33 am

File: Podfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ inhibit_all_warnings!
 #       the continuous build server.
 
 plugin 'cocoapods-keys', {
-    :project => "Artsy",
+    :project => "sdasdasdasdassdsad",
```
Comment (ashfurrow) [inline Podfile]: Gotta revert this ^

**Comment by orta** [comment]
I'm now confident in this PR 👍

**Comment by orta** [comment]
Triggered a CI without cache

File: Podfile [hunk 2]
```diff
@@ -77,19 +77,15 @@ target 'Artsy' do
...
 
   # Artsy Spec repo stuff
+  pod 'Artsy+UIFonts', :git => "https://github.com/artsy/Artsy-OSSUIFonts.git", :branch => "v3"
   pod 'Artsy-UIButtons', :git => 'https://github.com/artsy/Artsy-UIButtons.git'
-  pod 'Artsy+UIColors'
-  pod 'Artsy+UILabels'
-  pod 'Extraction'
+  pod 'Artsy+UIColors' 
+  pod 'Artsy+UILabels', :git => 'https://github.com/artsy/Artsy-UILabels.git',  :branch => "32"
+  pod 'Extraction', :git => 'https://github.com/artsy/Extraction.git',  :branch => "122"
```
Comment (alloy) [inline Podfile]: Why are these branches needed?

Comment (orta) [inline Podfile]: good point, let's get them shipped.

**Comment by orta** [comment]
Alright, that's all them

**Comment by alloy** [comment]
So much nice 👏

**Pull request merged**